tests/transport_test.go

```diff
@@ -0,0 +1,118 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+
+	"k8s.io/api/core/v1"
+)
+
+var _ = Describe("Transport Tests", func() {
+
+	const (
+		annEndpoint = "cdi.kubevirt.io/storage.import.endpoint"
+		annSecret   = "cdi.kubevirt.io/storage.import.secretName"
+
+		fileHostName   = "cdi-file-host" // deployment and service name
+		fileHostNs     = "kube-system"   // deployment and service namespace
+		fileHostUname  = "admin"         // http and s3 username
+		fileHostPasswd = "password"      // http and s3 password
+
+		httpPortName = "http"
+		s3PortName   = "s3"
+
+		targetFile = "tinyCore.iso"
+		bucket     = "images"
+	)
+
+	var (
+		httpEp, s3Ep, ns string
+		fileHostService  *v1.Service // persistent file host service, routes to http and s3 ports
+	)
+
+	f, err := framework.NewFramework("", framework.Config{false})
+	if err != nil {
+		Fail(errors.Wrap(err, "Failed to create test framework").Error())
+	}
+
+	c, err := f.GetKubeClient()
+	if err != nil {
```

Comment:
I guess the same `...OrDie` comment is valid here too...

Reply:
I don't really know why the clients aren't allocated by NewFramework... it has to create one internally anyway to generate the namespaces.

Reply:
They are but I forgot that. See [BeforeEach](https://github.com/kubevirt/containerized-data-importer/blob/master/tests/framework/framework.go#L126)

Reply:
We just merged a PR that fixes that.